README.md

```diff
@@ -1,8 +1,9 @@
 <h1 align="center"><img src="docs/images/readme_logo.png" width="500" alt="MsQuic logo"/></h1>
 
 [![Perf Dashboard](https://img.shields.io/static/v1?label=Performance&message=Dashboard&color=blue)](https://microsoft.github.io/msquic/)
-[![Build Status](https://img.shields.io/github/actions/workflow/status/microsoft/msquic/build.yml?label=Build)](https://github.com/microsoft/msquic/actions/workflows/build.yml)
-[![Test Status](https://img.shields.io/github/actions/workflow/status/microsoft/msquic/test.yml?label=Test)](https://github.com/microsoft/msquic/actions/workflows/test.yml)
+[![Build Status](https://img.shields.io/github/actions/workflow/status/microsoft/msquic/build.yml?query=branch%3Amain&label=Build)](https://github.com/microsoft/msquic/actions/workflows/build.yml?query=branch%3Amain)
+[![Test Status](https://img.shields.io/github/actions/workflow/status/microsoft/msquic/test.yml?query=branch%3Amain&label=Test)](https://github.com/microsoft/msquic/actions/workflows/test.yml?query=branch%3Amain)
+[![Stress Status](https://img.shields.io/github/actions/workflow/status/microsoft/msquic/stress.yml?query=branch%3Amain&label=Test)](https://github.com/microsoft/msquic/actions/workflows/stress.yml?query=branch%3Amain)
```

Comment:
This shows as `Test` still, i.e. there are two Test badges side by side

Reply:
Good catch